schemas/2015-08-01/Microsoft.Compute.json

```diff
@@ -192,7 +192,12 @@
             { "$ref": "#/definitions/genericExtension" },
             { "$ref": "#/definitions/iaaSDiagnostics" },
             { "$ref": "#/definitions/iaaSAntimalware" },
-            { "$ref": "#/definitions/customScriptExtension" }
+            { "$ref": "#/definitions/customScriptExtension" },
+            { "$ref": "#/definitions/customScriptForLinux"},
```

Comment:
What exactly are these tests doing? I've added one for each extension added, but the existing tests are using old apiversions that wouldn't support a GET?